responsibleai/responsibleai/_managers/counterfactual_manager.py

```diff
@@ -185,48 +185,31 @@ def compute(self):
         for cf_config in self._counterfactual_config_list:
             if not cf_config.is_computed:
                 cf_config.is_computed = True
-                try:
-                    dice_explainer = self._create_diceml_explainer(
-                        method=cf_config.method,
-                        continuous_features=cf_config.continuous_features)
-
-                    X_test = self._test.drop([self._target_column], axis=1)
-
-                    counterfactual_obj = \
-                        dice_explainer.generate_counterfactuals(
-                            X_test, total_CFs=cf_config.total_CFs,
-                            desired_class=cf_config.desired_class,
-                            desired_range=cf_config.desired_range)
-
-                    cf_config.counterfactual_obj = \
-                        counterfactual_obj
-                except Exception as e:
-                    cf_config.has_computation_failed = True
-                    cf_config.failure_reason = str(e)
-
-    def get(self, failed_to_compute=False):
-        """Return the computed counterfactual examples objects or failure reason.
-
-        :param failed_to_compute: Get the failure reasons if counterfactual
-                                  examples failed to compute.
-        :type failed_to_compute: bool
+                dice_explainer = self._create_diceml_explainer(
+                    method=cf_config.method,
+                    continuous_features=cf_config.continuous_features)
+
+                X_test = self._test.drop([self._target_column], axis=1)
+
+                counterfactual_obj = \
+                    dice_explainer.generate_counterfactuals(
+                        X_test, total_CFs=cf_config.total_CFs,
+                        desired_class=cf_config.desired_class,
+                        desired_range=cf_config.desired_range)
+
+                cf_config.counterfactual_obj = \
+                    counterfactual_obj
+
+    def get(self):
+        """Return the computed counterfactual examples
         """
-        if not failed_to_compute:
-            counterfactual_obj_list = []
-            for counterfactual_config in self._counterfactual_config_list:
-                if counterfactual_config.is_computed and \
-                        not counterfactual_config.has_computation_failed:
-                    counterfactual_obj_list.append(
-                        counterfactual_config.counterfactual_obj)
-            return counterfactual_obj_list
-        else:
-            failure_reason_list = []
-            for counterfactual_config in self._counterfactual_config_list:
-                if counterfactual_config.is_computed and \
-                        counterfactual_config.has_computation_failed:
-                    failure_reason_list.append(
-                        counterfactual_config.failure_reason)
-            return failure_reason_list
```

Comment:
The get() method returns the counterfactual object or the reson why the computation failed. Why this removed?

Reply:
This is the issue all the error get hidden with this get, user will not notice when the counter factual fail

Reply:
As I mentioned in the above comment, we could throw a warning that the computation failed as some reason. The error is not hidden. The error message is recorded in the class and user can access this error message. I am not able to understand in what capacity in the user not able to know that the counterfactual computation failed.